### ISSUE TITLE
English numbers in month view when locale changes

### ISSUE THREAD

**Issue by Lithium-HD** [wdullaer/MaterialDateTimePicker#325]
the problem is my app has two languages and when I set app's locale to non-English locale (like "fa" for Persian language), the numbers in calendar change. look at screen shots before and after changing the locale. how can I have the calendar in English locale only
I have set fontFamily for every textViews in layout files but didn't solve the problem.
![screenshot_2017-03-06-12-50-14](https://cloud.githubusercontent.com/assets/14963386/23603918/39c19bee-026c-11e7-9504-b478f8a9b1f0.png)
![screenshot_2017-03-06-12-50-33](https://cloud.githubusercontent.com/assets/14963386/23603919/39c316d6-026c-11e7-9ea9-53c37338a84d.png)

**Comment by wdullaer** [comment]
The picker tries to adapt to the locale of the device.
I'm amazed that the Month and Week labels do not change (they really should).

Someone actually made a fork of this library to support the Persian calendar system, maybe that is useful for you
https://github.com/wdullaer/MaterialDateTimePicker/issues/54

**Comment by Lithium-HD** [comment]
thanks for your reply, I have already mixed your library with a PersianCalendar. the reason that the month and week labels are changed is that I have set every textView's font-Family to sans-serif. maybe that's why :) I also tried to change day numbers too but I couldn't do that. anyway I used a dirty way to get over this issue (I set the locale to "en" before opening the calendar, and set it back to what is was after dismissing it)
I am in a rush, maybe I find a better way for this later.
